conan/recipes/example/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+  "v0.0.0.1":
```

Comment:
Shouldn't that be 0.0.1? (aren't we using semantic versioning? https://semver.org/)

Reply:
this would apply above as well

Reply:
If we want to require semver, I suggest moving that discussion over to https://github.com/beman-project/beman/pull/18.

As to this PR, I picked `0.0.0.1` because we don't have an actual release of the example repo yet, and `0.0.0.1` sorts really low. It's entirely a placeholder for now.

As to technical requirements, [Conan has docs to consider](https://docs.conan.io/2/tutorial/versioning/version_ranges.html), but it supports many options.

We *don't* want exactly semantic versioning for these version in these files, though. We need to be able to model multiple iterations on the packaging of version `X.Y.Z` of a specific library that was released in a repo external to this repo (i.e., all of them). This is because it's common enough to update or fix packaging details without wanting or needing to create a new version of the library being packaged.

So you might have a packaging bug in `foobar/X.Y.Z` where a few files weren't installed correctly. If it were instead versioned as `foobar/X.Y.Z-0`, you could reasonably bump to `foobar/X.Y.Z-1` to indicate the underlying source code hasn't changed but the packaging did.

I'm not familiar with if Conan has a popular convention for how to model this requirement. I'll poke some Conan folks for advice.

Reply:
Note on requiring semver, the scnlib repo already has source releases and a versioning convention. One of the tradeoffs in taking external repos as is is the need to play nice with whatever project conventions they come up with, including whether they use semver or something else.

Reply:
I consulted with a Conan core developer. It seems fine to use a version string that doesn't exactly match the tags on the upstream repo, so I made the proposed change here.